Orange/widgets/data/owpaintdata.py

```diff
@@ -905,18 +905,18 @@ def _init_ui(self):
             indBox, self, "brushRadius", minValue=1, maxValue=100,
             createLabel=False
         )
-        form.addRow("Radius", slider)
+        form.addRow("Radius:", slider)
 
         slider = gui.hSlider(
             indBox, self, "density", None, minValue=1, maxValue=100,
             createLabel=False
         )
 
-        form.addRow("Intensity", slider)
+        form.addRow("Intensity:", slider)
 
         gui.rubber(self.controlArea)
         gui.auto_commit(self.left_side, self, "autocommit",
-                        "Send", "Send on change")
+                        "Send")
```

Comment:
I prefer the old auto send message. Before, it said "Send on change", now it is "Auto Send Is On" -- this is the default message set by `gui.auto_commit` function. This "auto XXX is on" is too technical, so I almost always overrode it when I was introducing the `auto_commit` function.

I suggest though that we change the default in `gui.auto_commit` to `label.title() + "on change"`. Note that I wouldn't use capital letters; although it's technically a button, it's always disabled, and, more importantly, the text is not written as a "command" (e.g. "Apply", "Do This") but rather as a message.